torch/_inductor/lowering.py

```diff
@@ -2249,6 +2249,9 @@ def is_aligned(x):
 make_fallback(aten.frexp)
 make_fallback(aten.geqrf)
 make_fallback(aten.histc)
+make_fallback(aten.i0)
+make_fallback(aten.igamma, warn=False)
+make_fallback(aten.igammac, warn=False)
 make_fallback(aten.isin)
```

Comment:
Looks like a bad merge conflict resolution
```suggestion
```

Reply:
Ah my bad. I saw the additions and forgot that I should be deleting this line, not adding it.